source/draw/gpu/GpuCanvas.ooc

```diff
@@ -55,4 +55,9 @@ GpuCanvasYuv420Semiplanar: class extends GpuSurface {
 		this _target y canvas clear()
 		this _target uv canvas clear()
 	}
+	draw: override func ~mesh (image: Image, mesh: GpuMesh) {
```

Comment:
We might want to support RasterImages. In this case it's just a matter of creating a temporary GpuImage if the input image is a RasterImage. Since this function is (and probably will always be) a special case only used in the rolling shutter distortion correction I've been lazy regarding these matters. It's up to you if you think we should raise exceptions etc.